graylog2-server/src/main/java/org/graylog2/indexer/messages/Indexable.java

```diff
@@ -38,11 +39,21 @@ public interface Indexable {
      * The message id is represented as a {@link de.huxhorn.sulky.ulid.ULID}
      */
     String getMessageId();
+
     long getSize();
+
     DateTime getReceiveTime();
-    Map<String, Object> toElasticSearchObject(ObjectMapper objectMapper,@Nonnull final Meter invalidTimestampMeter);
+
+    Map<String, Object> toElasticSearchObject(ObjectMapper objectMapper, @Nonnull final Meter invalidTimestampMeter);
+
     DateTime getTimestamp();
 
+    default byte[] serialize(SerializationContext context) throws JsonProcessingException {
```

Comment:
How about throwing an `IOException` instead of the JSON-specific `JsonProcessingException`? That makes it a bit more generic.